client/src/assets/css/home.css

```diff
@@ -15,4 +15,16 @@
 .searchbar-container {
     margin-top: 31px;
     margin-bottom: 27px;
+}
+
+@media only screen and (max-width: 540px) {
```

Comment:
just wondering could you provide a bit more information on why only these two width are required ?

Reply:
On this is from doing browser testing. I can do a quick demo of why I chose these numbers